### PR TITLE
support differenct platform binary

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

File: pyzoo/dev/release/release.sh [hunk 1]
```diff
@@ -42,8 +42,11 @@ if [ "$platform" ==  "mac" ]; then
     verbose_pname="macosx_10_11_x86_64"
 elif [ "$platform" == "linux" ]; then
     echo "Building Analytics Zoo for linux system"
-    dist_profile="-P without_bigdl $profiles"
+    dist_profile="-P linux -P without_bigdl $profiles"
     verbose_pname="manylinux1_x86_64"
+elif [ "$platform" == "windows" ]; then
+    echo "Building Analytics Zoo for windows system"
+    dist_profile="-P windows -P without_bigdl $profiles"
```
Comment (hkvision) [inline pyzoo/dev/release/release.sh]: do we need verbose_pname for windows? (Line 46 for Linux)

**Comment by hkvision** [comment]
LGTM besides the comments above.

**Comment by hkvision** [comment]
Then we need to update pip release job to add windows platform.

**Comment by yangw1234** [comment]
jenkins pass http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/

**Pull request merged**